android/CouchbaseLite/src/main/java/com/couchbase/lite/ConsoleLogger.java

```diff
@@ -30,22 +30,20 @@
  */
 public final class ConsoleLogger implements Logger {
     private LogLevel _level = LogLevel.WARNING;
-    private ArrayList<LogDomain> _domains = new ArrayList<>();
+    private EnumSet<LogDomain> _domains = EnumSet.allOf(LogDomain.class);
```

Comment:
I think we should do `private EnumSet<LogDomain> _domains = EnumSet.of(LogDomain.ALL);`